### ISSUE TITLE
Alter windows unit test to remove pagefile or configure it correctly

### ISSUE THREAD

**Issue by mikehardy** [ankidroid/Anki-Android#15372]
### Checked for duplicates?

- [X] This issue is not a duplicate

### What are the steps to reproduce this bug?

Examine a windows unit test run and see this:

```
Error: Exception calling "SetPageFileSize" with "3" argument(s): "The operation completed successfully"
At D:\a\_actions\al-cheb\configure-pagefile-action\v1.3\scripts\SetPageFileSize.ps1:193 char:1
+ [Util.PageFile]::SetPageFileSize($minimumSize, $maximumSize, $diskRoo ...
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [], MethodInvocationException
    + FullyQualifiedErrorId : Win32Exception
```

### Expected behaviour

Successful completion not an error

### Actual behaviour

Error message above

### Debug info

```text
Current main as of this typing
```

### (Optional) Anything else you want to share?

I believe switching the root from D: to C: will fix it as discussed here (and with verification of the fix appearing to work on a repo using it, that linked to that issue, upon examination of their run logs) - https://github.com/al-cheb/configure-pagefile-action/issues/18#issuecomment-1865226294

### Research

- [X] I am reporting a bug specific to AnkiDroid (Android app)
- [X] I have checked the [manual](https://ankidroid.org/docs/manual.html) and the [FAQ](https://github.com/ankidroid/Anki-Android/wiki/FAQ) and could not find a solution to my issue
- [ ] (Optional) I have confirmed the issue is not resolved in the latest alpha release ([instructions](https://docs.ankidroid.org/manual.html#betaTesting))

**Comment by mikehardy** [comment]
Also, hopefully my fix of that action from node 16 to node 20 will quiet some CI warnings https://github.com/al-cheb/configure-pagefile-action/pull/20

**Comment by mikehardy** [comment]
still waiting on my upstream PR to work through process there before I turn focus back here

**Comment by mikehardy** [comment]
upstream merged as 1.4. I think I prefer to have a pagefile than not, and have any windows CI issue be a performance issue vs a crash issue, so I'll go for "configure it correctly"
